mdn/kuma
# 7172 sendinblue sync username

Fixes #7172

## What it does
- extends newsletter system check to create the `USERNAME` attribute
- extends the `export_to_sendinblue` command, as well as the newsletter tasks to populate the new attribute
- adds a signal handler for when the username has changed, calls it appropriately and adds a receiver that triggers a sendinblue sync for subscribed users
  - A special note for @peterbe when working on #7246: This adds another detail to port over into the new endpoint (checking for username change, and sending the appropriate signal)

## How to test it
- start kuma
- 🔎 check https://my.sendinblue.com/users/list and marvel at the new attribute being created
- 🔎 run `docker-compose exec web python manage.py export_to_sendinblue` and be shook by the new attribute being populated (again, by looking at https://my.sendinblue.com/users/list)
- 🔎 change a newsletter-subscribed user's username through the user_edit form and once again experience bliss, as that user's `USERNAME` will be updated at https://my.sendinblue.com/users/list
